src/app/validators/components/validator-status/validator-status.component.html

```diff
@@ -1,11 +1,19 @@
 <svg-icon
   [key]="validatorStatusName.toLowerCase()"
   size="lg"
+  *appBreakpoint="shrinkBreakpoint"
```

Comment:
*appBreakpoint="shrinkBreakpoint; else template" instead of `<ng-container> `below